packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -279,6 +285,35 @@ export const Webchat = forwardRef((props, ref) => {
   const resendUnsentInputs = async () =>
     props.resendUnsentInputs && props.resendUnsentInputs()
 
+  // Load styles stored in window._botonicInsertStyles by Webpack
+  useComponentWillMount(() => {
+    if (!window._botonicInsertStyles || !window._botonicInsertStyles.length)
+      return
```

Comment:
Even if we don't have styles in `_botonicInsertStyles` we want to load the `emoji-picker-react` styles, right? If so, this early return should be placed after injecting the emoji-picker-react styles.

Reply:
true... maybe I'll just return early if `shadowDOM` is false